### DISCUSSION TITLE
Reduced-complexity modes of FATES

### DISCUSSION THREAD

**Discussion by evalieungh** [NorESMhub/noresm-land-sites-platform#176] [General]
I just discovered that Satellite Phenology mode of FATES seems to work in the platform already. All you have to do is enter `use_fates_sp = .true.` into the Advanced box of the CLM tab in the GUI when creating a new case!

This worked for me in a short test simulation, but ideally we would need to test it a bit more, write this into the documentation, and check if the model needs to be updated for the various reduced-complexity modes to work properly. 

Note that turning off PFTs in the FATES tab simultaneously causes the model to crash.

**Comment by rosiealice** [comment]
Great! 

A few things:

What in effect does the pft tab do? 

You could also try nocomp and fixed_biogeog, which work with
Use_fates_nocomp and use_fates_fixed_biogeog from what I remember.

**Reply by lasseke** [reply]
The PFT tab is just the part of the GUI responsible for manipulating the FATES parameter file. You can change some PFT-specific parameters, mainly for demonstration purposes, and disable PFTs for the default fully competitive mode.
